spark-jobs/pom.xml

```diff
@@ -181,7 +187,9 @@
                             <include>*:*</include>
                         </includes>
                         <excludes>
-                            <exclude>org.apache.spark:*</exclude>
+<!--  org.apache.spark:* should not be excluded. Otherwise, the spark-submit would have to be called with
+spark-submit -(remove this)-packages org.apache.spark:spark-sql-kafka-0-10_2.11:2.2.0
```

Comment:
Not sure about the `(remove this)` comment in the middle of an option.

Reply:
`--` would break the comment

Reply:
I have looked around. This comes to how stupid XML is. People seem to use `-\-` to battle it. I will leave it up to you.